debian/watch

```diff
@@ -0,0 +1,2 @@
+# The project has no releases, yet, so I am incrementing the patch
+# version manually and using things directly from git.
```

Comment:
There's no real easy way to do that inside a watch file, unfortunately.  This isn't a big deal - the file being present just shuts up one of our lint programs.